ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractMiningCoordinator.java

```diff
@@ -215,4 +232,28 @@ public void setExtraData(final Bytes extraData) {
 
   protected abstract boolean newChainHeadInvalidatesMiningOperation(
       final BlockHeader newChainHeadHeader);
+
+  @Override
+  public Map<String, SealerInfo> getSealerInfos() {
```

Comment:
I think it would be better to move all the logic around submitting hashrate and storing the sealerInfo down into the EthashMiningCoordinator.  IBFT and Clique will never use these.

Reply:
Done